sway-lib-std/src/storage.sw

```diff
@@ -66,12 +62,12 @@ use ::result::Result;
         let result_ptr = stack_ptr();
 
         while size_left > 32 {
-            // Read 4 words (32 bytes) at a time using `srwq`
+            // Read 4 words (32 bytes) at a time
             let current_pointer = stack_ptr();
-            asm(k: local_key, v: current_pointer) {
+            asm() {
                 cfei i32;
-                srwq v k;
             };
```

Comment:
Next is an `__alloca()` intrinsic to wrap `CFEI` since it's a bit weird to grab the stack pointer with  a call but then allocate some space with an ASM block.  I guess it could go in a library function for now...